src/components/Footer.js

```diff
@@ -0,0 +1,14 @@
+import React from "react"
+
+const Footer = () => {
+  return (
+    <section className="about">
+      UtahJS is a 501(c)(3) organization and contributions are tax deductible.
```

Comment:
Can we wrap the text in a p tag for semantics/accessibility.

Reply:
Yes, I was considering that was not best a11y practice, but decided just to copy the section tag like it was on the old site.  Perhaps we should use  footer/p tags?
https://www.w3schools.com/tags/tag_footer.asp